src/visualizations/VaccinationChart.fs

```diff
@@ -26,6 +26,7 @@ type DisplayType =
         | ByManufacturer -> chartText "byManufacturer"
 
 let AllVaccinationTypes = [
+    "janssen",     "#019cdc"
```

Comment:
This blue is from https://www.janssen.com/slovenia/ but seems too similar to Pfizer. Other suggestions?